lib/client/client.js

```diff
@@ -298,6 +298,46 @@ Client.prototype.bind = function bind (name,
   return this._send(req, [errors.LDAP_SUCCESS], null, callbackWrapper, _bypass)
 }
 
+/**
+ * Performs a sasl (NTLM/GSSAPI) authentication against the server.
+ *
+ * @param {String} token the token from the authentication http header.
+ * @param {Control} controls (optional) either a Control or [Control].
+ * @param {Function} callback of the form f(err, res).
+ * @throws {TypeError} on invalid input.
+ */
+Client.prototype.saslBind = function saslBind (token,
```

Comment:
Are we able to add a corresponding test for this in `client.test.js?

Reply:
Sure